packages/organisations-api/get-organisation-stops/index.ts

```diff
@@ -0,0 +1,214 @@
+                disruptionEndDate: disruption.disruptionEndDate,
+                disruptionEndTime: disruption.disruptionEndTime,
+                disruptionNoEndDateTime: disruption.disruptionNoEndDateTime,
+                disruptionRepeats: disruption.disruptionRepeats,
+                disruptionRepeatsEndDate: disruption.disruptionRepeatsEndDate,
+            });
+
+            const maxEndDate = getMaxEndDate(mergedValidities);
+
+            return startDate.isSameOrBefore(currentDate) && (maxEndDate ? currentDate.isBefore(maxEndDate) : true);
+        });
+
+        const [stops, services] = await Promise.all([
+            getStopsFromDisruptions(activeAndPublishedDisruptions),
+            getServicesFromDisruptions(activeAndPublishedDisruptions),
+        ]);
+
+        const routesForMaps: ServiceGeoJSON[] = [];
+        await Promise.all(
+            services.map(async (service) => {
+                const routesData = await fetchServiceRoutes(service.id, logger);
+
+                const [inboundRoute, outboundRoute] = await Promise.all([
+                    fetchRoute(routesData?.inbound ?? []),
+                    fetchRoute(routesData?.outbound ?? []),
+                ]);
+
+                const inboundGeoJSON: ServiceGeoJSON = getGeoJson(inboundRoute, service);
+
+                const outboundGeoJSON: ServiceGeoJSON = getGeoJson(outboundRoute, service);
+
+                if (inboundRoute && inboundRoute.length > 0) routesForMaps.push(inboundGeoJSON);
+                if (outboundRoute && outboundRoute.length > 0) routesForMaps.push(outboundGeoJSON);
+            }),
+        );
+
+        return { stops, services: routesForMaps };
+    } catch (e) {
+        if (e instanceof Error) {
+            logger.error(e);
+
+            throw e;
+        }
+
+        throw e;
+    }
+};
+
+export const main = async (event: APIGatewayEvent): Promise<{ statusCode: number; body: string }> => {
+    try {
+        logger.options.dev = process.env.NODE_ENV !== "production";
+        logger.options.debug = process.env.ENABLE_DEBUG_LOGS === "true" || process.env.NODE_ENV !== "production";
+
+        logger.options.meta = {
+            id: randomUUID(),
+        };
+        logger.info("Starting get organisation stops data...");
+
+        const { ORGANISATIONS_TABLE_NAME: orgTableName } = process.env;
+
+        if (!orgTableName) {
+            throw new Error("Dynamo table names not set");
+        }
+
+        const orgId = event?.pathParameters?.id;
+
+        if (!orgId) {
+            throw new Error("An organisation ID must be provided");
+        }
+
+        const stopsAndServicesMapData = await getOrganisationStops(orgId);
+
+        logger.info("Successfully retrieved organisations DynamoDB...");
+
+        return {
+            statusCode: 200,
+            body: JSON.stringify(stopsAndServicesMapData),
+        };
+    } catch (e) {
+        if (e instanceof Error) {
+            logger.error(e);
```

Comment:
could we have a message here too like 'Failed to get organisation stops and routes for services data'

Reply:
done